src/hardware/scpi-pps/profiles.c

```diff
@@ -1853,6 +1690,42 @@ SR_PRIV const struct scpi_pps pps_profiles[] = {
 		hp_6630b_update_status,
 	},
 
+	/* Keysight E36311A; NOT TESTED*/
+	{ "Keysight", "E36311A", SCPI_DIALECT_KEYSIGHT_E36300A, 0,
+		ARRAY_AND_SIZE(keysight_e36300a_devopts),
+		ARRAY_AND_SIZE(keysight_e36300a_devopts_cg),
+		ARRAY_AND_SIZE(keysight_e36311a_ch),
+		ARRAY_AND_SIZE(keysight_e36300a_cg),
+		keysight_e36300a_cmd,
+		.probe_channels = NULL,
+		.init_acquisition=NULL,
+		.update_status=NULL,
+	},
+
+	/* Keysight E36312A */
+	{ "Keysight", "E36312A", SCPI_DIALECT_KEYSIGHT_E36300A, 0,
+		ARRAY_AND_SIZE(keysight_e36300a_devopts),
+		ARRAY_AND_SIZE(keysight_e36300a_devopts_cg),
+		ARRAY_AND_SIZE(keysight_e36312a_ch),
+		ARRAY_AND_SIZE(keysight_e36300a_cg),
+		keysight_e36300a_cmd,
+		.probe_channels = NULL,
+		.init_acquisition=NULL,
+		.update_status=NULL,
+	},
+
+	/* Keysight E36313A; NOT TESTED*/
+	{ "Keysight", "E36313A", SCPI_DIALECT_KEYSIGHT_E36300A, 0,
+		ARRAY_AND_SIZE(keysight_e36300a_devopts),
+		ARRAY_AND_SIZE(keysight_e36300a_devopts_cg),
+		ARRAY_AND_SIZE(keysight_e36313a_ch),
+		ARRAY_AND_SIZE(keysight_e36300a_cg),
+		keysight_e36300a_cmd,
+		.probe_channels = NULL,
+		.init_acquisition=NULL,
+		.update_status=NULL,
+	},
+
```

Comment:
FYI, I left those in because all other models have these or similar "= NULL" entries as well, e.g. https://github.com/sigrokproject/libsigrok/blob/master/src/hardware/scpi-pps/profiles.c#L1520